### DISCUSSION TITLE
Can I use Verdaccio private repo AND publish to NPM public repo?

### DISCUSSION THREAD

**Discussion by lestan** [verdaccio/verdaccio#4549] [Q&A] [answered]
I'm successfully using verdaccio to host a local private repository and am able to publish my package into it.

However, now that I'm ready to make my package public, I'd like to also publish it into the npmjs public repo.  It's unclear to me how exactly to do that and I haven't found any clear instructions.

Is this possible?  Can someone point me to a guide or help with instructions.

Thanks!!

Les

**Comment by juanpicado** [comment, marked as answer]
> Is this possible? Can someone point me to a guide or help with instructions.

No, and will never be. The main reason is would create concerns to those use verdaccio that could leak their private packages. If you want to publish that's a package manager work, verdaccio is just a private registry and all it does is store packages either private or those fetched from upstream.
